Gems/PhysX/Code/Include/PhysX/Joint/PhysXJointRequestsBus.h

```diff
@@ -47,6 +53,11 @@ namespace PhysX
         //! Sets maximum motor force.
         //! @param force in Newtons (for prismatic joint) or Newton-meters (for hinge joint).
         virtual void SetMaximumForce(float force) = 0;
+
+        //! Get forces (force vector and torque vector) from joint.
+        //! @returns pair of 3d vectors : force in Newtons and torque in Newton-meters.
+        virtual AZStd::pair<AZ::Vector3, AZ::Vector3> GetForces() const = 0;
```

Comment:
suggestion: Making a new little struct (`struct ForcePair`?) might be nice so the fields can be named (and commented) as opposed to using a `pair` (as `first` and `second` are not always that helpful 🙈). Just a suggestion, feel free to ignore if you think this is sufficient.

Reply:
+1